src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -3,11 +3,11 @@
   <ItemGroup>
     <RuntimeGroup Include="unix">
       <Parent>any</Parent>
-      <Architectures>x64;x86;arm;armv6;armel;arm64;loongarch64;mips64;s390x;ppc64le</Architectures>
+      <Architectures>x64;x86;arm;armv6;armel;arm64;loongarch64;riscv64;mips64;s390x;ppc64le</Architectures>
```

Comment:
Nit: `riscv64` is the last one everywhere else. It would be nice to have it last here as well for consistency.

Reply:
Fixed!